### PR TITLE
Milestone4

### PR DESCRIPTION
**Pull request by hemantso**
- logic folder created
- operate module to calculate total
- calculate module to pass the calculator data object and button name

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @hemant-soni-vst-au4 
### You have done well so far :clap: However, you have an issue to fix:-
- [x] Make the demo link display the calculator, currently, it displays an error.
### Fix the issue and submit for another code review. Good luck.
- Happy coding!

**Comment by hemantso** [comment]
- live link is working now.
-  Styling of the calculator will be done in milestone 5

**Comment by sumancrest0001** [comment]
**Status: Approved :clap: :100: 
Congratulations @hemant-soni-vst-au4,**
Thank you for implementing the change requested by the previous TSE. Now, the live link is working correctly.

Happy coding !!!

Before moving to the next milestone, please consider the following points.

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      next *= -1;
+      break;
+    case '=':
+    case '%':
+    case '+':
+    case '-':
+    case '÷':
+    case 'x':
+      total = operate(total, next, operation);
+      break;
```
Comment (sumancrest0001) [inline src/logic/calculate.js:18]: [Optional] It would have been nice if you added logic for the operations as well.

**Review by sumancrest0001 — APPROVED** [review]
[Optional] You have assigned the same key for all the buttons in the ButtonPanel file which resulted in an error. We can fix that.

**Pull request merged**